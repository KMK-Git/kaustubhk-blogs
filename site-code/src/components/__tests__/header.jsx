import React from 'react';
import renderer from 'react-test-renderer';

import Header from '../header';

const { act } = renderer;

describe('Header', () => {
  it('renders correctly', async () => {
    let tree;
    act(() => {
      tree = renderer
        .create(<Header />);
    });
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
