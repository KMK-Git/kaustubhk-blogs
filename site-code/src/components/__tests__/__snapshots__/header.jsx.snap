// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
[
  <style
    dangerouslySetInnerHTML={
      {
        "__html": ".css-1lrg2we-MuiGrid-root{box-sizing:border-box;display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-box-flex-wrap:wrap;-webkit-flex-wrap:wrap;-ms-flex-wrap:wrap;flex-wrap:wrap;width:100%;-webkit-flex-direction:row;-ms-flex-direction:row;flex-direction:row;}@media (min-width:0px){.css-1lrg2we-MuiGrid-root{margin-top:-24px;}.css-1lrg2we-MuiGrid-root>.MuiGrid-item{padding-top:24px;}}@media (min-width:0px){.css-1lrg2we-MuiGrid-root{width:calc(100% + 24px);margin-left:-24px;}.css-1lrg2we-MuiGrid-root>.MuiGrid-item{padding-left:24px;}}",
      }
    }
    data-emotion="css 1lrg2we-MuiGrid-root"
  />,
  <div
    align="center"
    className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3 css-1lrg2we-MuiGrid-root"
  >
    <style
      dangerouslySetInnerHTML={
        {
          "__html": ".css-zbaamk-MuiGrid-root{box-sizing:border-box;margin:0;-webkit-flex-direction:row;-ms-flex-direction:row;flex-direction:row;-webkit-flex-basis:100%;-ms-flex-preferred-size:100%;flex-basis:100%;-webkit-box-flex:0;-webkit-flex-grow:0;-ms-flex-positive:0;flex-grow:0;max-width:100%;}@media (min-width:600px){.css-zbaamk-MuiGrid-root{-webkit-flex-basis:25%;-ms-flex-preferred-size:25%;flex-basis:25%;-webkit-box-flex:0;-webkit-flex-grow:0;-ms-flex-positive:0;flex-grow:0;max-width:25%;}}@media (min-width:900px){.css-zbaamk-MuiGrid-root{-webkit-flex-basis:25%;-ms-flex-preferred-size:25%;flex-basis:25%;-webkit-box-flex:0;-webkit-flex-grow:0;-ms-flex-positive:0;flex-grow:0;max-width:25%;}}@media (min-width:1200px){.css-zbaamk-MuiGrid-root{-webkit-flex-basis:25%;-ms-flex-preferred-size:25%;flex-basis:25%;-webkit-box-flex:0;-webkit-flex-grow:0;-ms-flex-positive:0;flex-grow:0;max-width:25%;}}@media (min-width:1536px){.css-zbaamk-MuiGrid-root{-webkit-flex-basis:25%;-ms-flex-preferred-size:25%;flex-basis:25%;-webkit-box-flex:0;-webkit-flex-grow:0;-ms-flex-positive:0;flex-grow:0;max-width:25%;}}",
        }
      }
      data-emotion="css zbaamk-MuiGrid-root"
    />
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-3 css-zbaamk-MuiGrid-root"
    />
    <div
      align="center"
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-3 css-zbaamk-MuiGrid-root"
    >
      <a
        href="https://kaustubhk.com"
        rel="noopener noreferrer"
        style={
          {
            "textDecoration": "none",
          }
        }
        target="_blank"
      >
        <style
          dangerouslySetInnerHTML={
            {
              "__html": ".css-4pztnx-MuiCard-root{overflow:hidden;background-color:#feff9c;max-width:200px;max-height:50px;display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-flex-direction:row;-ms-flex-direction:row;flex-direction:row;margin-top:8px;-webkit-align-items:center;-webkit-box-align:center;-ms-flex-align:center;align-items:center;-webkit-box-pack:center;-ms-flex-pack:center;-webkit-justify-content:center;justify-content:center;}",
            }
          }
          data-emotion="css 4pztnx-MuiCard-root"
        />
        <style
          dangerouslySetInnerHTML={
            {
              "__html": ".css-k60sge-MuiPaper-root-MuiCard-root{background-color:#fff;color:rgba(0, 0, 0, 0.87);-webkit-transition:box-shadow 300ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;transition:box-shadow 300ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;border-radius:4px;box-shadow:0px 5px 5px -3px rgba(0,0,0,0.2),0px 8px 10px 1px rgba(0,0,0,0.14),0px 3px 14px 2px rgba(0,0,0,0.12);overflow:hidden;background-color:#feff9c;max-width:200px;max-height:50px;display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-flex-direction:row;-ms-flex-direction:row;flex-direction:row;margin-top:8px;-webkit-align-items:center;-webkit-box-align:center;-ms-flex-align:center;align-items:center;-webkit-box-pack:center;-ms-flex-pack:center;-webkit-justify-content:center;justify-content:center;}",
            }
          }
          data-emotion="css k60sge-MuiPaper-root-MuiCard-root"
        />
        <div
          className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation8 MuiCard-root css-k60sge-MuiPaper-root-MuiCard-root"
        >
          <style
            dangerouslySetInnerHTML={
              {
                "__html": ".css-46bh2p-MuiCardContent-root{padding:16px;}.css-46bh2p-MuiCardContent-root:last-child{padding-bottom:24px;}",
              }
            }
            data-emotion="css 46bh2p-MuiCardContent-root"
          />
          <div
            className="MuiCardContent-root css-46bh2p-MuiCardContent-root"
          >
            <style
              dangerouslySetInnerHTML={
                {
                  "__html": ".css-ag7rrr-MuiTypography-root{margin:0;font-family:"Roboto","Helvetica","Arial",sans-serif;font-weight:400;font-size:1.5rem;line-height:1.334;letter-spacing:0em;}",
                }
              }
              data-emotion="css ag7rrr-MuiTypography-root"
            />
            <div
              className="MuiTypography-root MuiTypography-h5 css-ag7rrr-MuiTypography-root"
            >
              My Homepage
            </div>
          </div>
        </div>
      </a>
    </div>
    <div
      align="center"
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-3 css-zbaamk-MuiGrid-root"
    >
      <a
        href="/"
        style={
          {
            "textDecoration": "none",
          }
        }
      >
        <div
          className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation8 MuiCard-root css-k60sge-MuiPaper-root-MuiCard-root"
        >
          <div
            className="MuiCardContent-root css-46bh2p-MuiCardContent-root"
          >
            <style
              dangerouslySetInnerHTML={
                {
                  "__html": ".css-9bxe7g-MuiTypography-root{margin:0;font-family:"Roboto","Helvetica","Arial",sans-serif;font-weight:400;font-size:1.5rem;line-height:1.334;letter-spacing:0em;padding:16px;padding-bottom:11px;}",
                }
              }
              data-emotion="css 9bxe7g-MuiTypography-root"
            />
            <div
              className="MuiTypography-root MuiTypography-h5 css-9bxe7g-MuiTypography-root"
            >
              All Blogs
            </div>
          </div>
        </div>
      </a>
    </div>
  </div>,
]
`;
